plugins/org/src/components/Cards/User/UserProfileCard/UserProfileCard.tsx

```diff
@@ -66,11 +75,27 @@ export const UserProfileCard = (props: { variant?: InfoCardVariants }) => {
     kind: 'Group',
   });
 
+  const infoCardAction = entityMetadataEditUrl ? (
+    <IconButton
+      aria-label="Edit"
+      title="Edit Metadata"
+      component={Link}
+      to={entityMetadataEditUrl}
+    >
+      <EditIcon />
+    </IconButton>
+  ) : (
+    <IconButton aria-label="Edit" disabled title="Edit Metadata">
+      <EditIcon />
+    </IconButton>
```

Comment:
Wonder if we should just hide it instead tbh? Reasoning for that being that if you don't have the ability to edit users in your org then this can potentially be confusing and waste a bit of real estate

Reply:
OK, cool, no refresh works for me. I've also updated this to hide the button if there is no edit annotation, I update the screenshot in the PR description as well. Thanks!